src/visual.ts

```diff
@@ -735,7 +755,8 @@ export class TableHeatMap implements IVisual {
         const width: number =
             viewport.width -
             this.margin.left -
-            this.margin.right;
+            this.margin.right -
+            2.1 * this.settingsModel.yAxisLabels.margin.value;
```

Comment:
What is 2.1?

Reply:
It should really be two since we want the margin on both sides. [86617b4](https://github.com/microsoft/powerbi-visuals-heatmap/pull/118/commits/86617b4b7b3532655abf196b1991a2854897d10d)